src/ocm/components/clusterConfiguration/review/ReviewValidations.tsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import {
+  Divider,
+  ExpandableSection,
+  Grid,
+  GridItem,
+  gridSpans,
+  Split,
+  SplitItem,
+} from '@patternfly/react-core';
+import {
+  CheckCircleIcon,
+  ExclamationCircleIcon,
+  ExclamationTriangleIcon,
+  InfoCircleIcon,
+} from '@patternfly/react-icons';
+import {
+  Cluster,
+  ClusterValidations,
+  DetailItem,
+  DetailList,
+  HostsValidations,
+  stringToJSON,
+  useFeatureSupportLevel,
+} from '../../../../common';
+import { useClusterWizardContext } from '../../clusterWizard/ClusterWizardContext';
+import { useOpenshiftVersions } from '../../../hooks';
+import { wizardStepNames } from '../../clusterWizard/constants';
+import {
+  ClusterWizardStepsType,
+  wizardStepsValidationsMap,
+  allClusterWizardSoftValidationIds,
+} from '../../clusterWizard/wizardTransition';
+import { ClusterFeatureSupportLevelsDetailItem } from '../../featureSupportLevels';
+import {
+  SupportLevelMemo,
+  getSupportLevelInfo,
+} from '../../featureSupportLevels/ReviewClusterFeatureSupportLevels';
+import {
+  global_success_color_100 as okColor,
+  global_info_color_100 as infoColor,
+  global_danger_color_100 as dangerColor,
+  global_warning_color_100 as warningColor,
+} from '@patternfly/react-tokens';
+import { useTranslation } from '../../../../common/hooks/use-translation-wrapper';
+
+import { ValidationsInfo as ClusterValidationsInfo } from '../../../../common/types/clusters';
+import { ValidationsInfo as HostValidationsInfo } from '../../../../common/types/hosts';
+
+const ValidationsDetailExpanded = ({ cluster }: { cluster: Cluster }) => {
```

Comment:
Could you please add the label for this validation?
`time-synced-between-host-and-service check failed`

![new-validation](https://user-images.githubusercontent.com/829045/208869327-eb9da61a-9f1d-41e7-93ba-152a3fbb8d8d.png)

Reply:
Personally I would prefer if validations were grouped by severity, eg. all host validations errors together, etc but this probably wasn't on the design.

Reply:
I think the same.

Reply:
Labels added.